.travis.yml

```diff
@@ -95,6 +101,26 @@ matrix:
 
     - python: 3.3
       env: TEST_DOCS=1 DJANGO=1.10 DATABASE_URL='sqlite://localhost/:memory:' MIGRATE_OPTION='--migrate'
+    - python: 3.4
+      env: TEST_DOCS=1 DJANGO=1.10 DATABASE_URL='sqlite://localhost/:memory:' MIGRATE_OPTION='--migrate'
+    - python: 3.5
+      env: TEST_DOCS=1 DJANGO=1.10 DATABASE_URL='sqlite://localhost/:memory:' MIGRATE_OPTION='--migrate'
+    - python: 2.7
+      env: TEST_DOCS=1 DJANGO=1.10 DATABASE_URL='sqlite://localhost/:memory:' MIGRATE_OPTION='--migrate'
+
+    - python: 3.3
```

Comment:
To me we can limit a lot the tests on different python versions: focusing on 2.7/3.5 with limited matrix on other versions